jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/SnackbarSection.java

```diff
@@ -0,0 +1,31 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.common.Snackbar;
+import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.common.TextField;
+
+public class SnackbarSection extends Section {
+
```

Comment:
In my opinion, we should only see two types of snack bars here, not fields, buttons etc:
public Snackbar basicSnackbar;
public Snackbar customSnackbar;